builder/amazon/common/ami_config.go

```diff
@@ -18,6 +18,8 @@ type AMIConfig struct {
 	AMISkipRegionValidation bool              `mapstructure:"skip_region_validation"`
 	AMITags                 map[string]string `mapstructure:"tags"`
 	AMIEnhancedNetworking   bool              `mapstructure:"enhanced_networking"`
+	ENASupport              bool              `mapstructure:"ena_support"`
```

Comment:
these should be prefixed with AMI (like the other options) so that it's obvious what config this is from. We embed the AMIConfig struct in builder configs, so its helpful to keep track of what struct it came from.

Reply:
That makes a lot of sense.  Will do.